src/MPDevice.cpp

```diff
@@ -1994,7 +1994,7 @@ bool MPDevice::addChildToDB(MPNode* parentNodePt, MPNode* childNodePt)
         else
         {
             /* If the login name is supposed to be before the one we're trying to insert */
-            if (tempNextChildNodePt->getLogin() > childNodePt->getLogin())
+            if (tempNextChildNodePt->getLogin().compare(childNodePt->getLogin()) > 0)
```

Comment:
The problem here is not to keep it simple, but make it correct. And the Qt doc explicitly says that QString::compare is not what a human would expect:

> Case sensitive comparison is based exclusively on the numeric Unicode values of the characters and is very fast, but is not what a human would expect. Consider sorting user-visible strings with localeAwareCompare().

http://doc.qt.io/qt-5/qstring.html#compare

A good solution is to use QCollator that allows more natural sorting of string containing numbers like:
blabla_1
blabla_2
blabla_10

Using compare or standard the list would be sorted as:
blabla_1
blabla_10
blabla_2

Which is wrong for us humans.